### ISSUE TITLE
Drag and drop support for new posts / thoughts

### ISSUE THREAD

**Issue by grand-lotus-iroh** [usememos/memos#2250]
### Is your feature request related to a problem?

**I think it would be awesome** if you could just CTRL + V from clipboard and upload that way too.

I would like to have the ability to upload pictures when creating them by dragging and dropping them, instead of having to manually select them through a file picker dialog. 

The feature is already half-baked and integrated, it's just that you have to click the picture icon to activate the drag and drop.

![image](https://github.com/usememos/memos/assets/1291364/ea03ea7c-80c7-4a81-b5f6-b070df4668b6)

### Describe the solution you'd like

When creating a new post (or thought), such as an image or a document, user can drag the file from the file explorer and drop it into the resource creation window, instead of having to manually select the file through a file picker dialog.

Adding clipboard support would be fantastic!

### Additional context

Benefits:

More intuitive and user-friendly experience.
Improved productivity and efficiency, as it saves time and effort compared to manually selecting files through a file picker dialog.

**Comment by boojack** [comment]
Ctrl C+V and drag&drop to upload resources is already supported in the editor area.

**Comment by grand-lotus-iroh** [comment]
That's not true @boojack — I just tested in Chromium browser Windows machine.

What editor area are you talking about? The default landing page where I can save a memo??

When attempting to paste a .jpg or .png file from my clipboard into the "thoughts" area using CTRL + V, it does not function as expected. Additionally, I experimented with dragging and dropping the image from File Explorer into the same area, but this approach also proved unsuccessful.

![image](https://github.com/usememos/memos/assets/1291364/857c3f10-6d8e-4d82-a016-50d4c9937683)

Clicking the image icon opens a modal popup window where I can drag and drop an image. However, this drag-and-drop functionality is limited to the "Choose a file" box within the modal, and it should ideally allow dropping an image anywhere on the entire screen. Additionally, using CTRL + V for pasting images does not function on this particular screen.

![image](https://github.com/usememos/memos/assets/1291364/47ea3f4a-87a6-4826-b8e8-da298144b430)
